src/Transformer.ts

```diff
@@ -36,9 +36,15 @@ export class Transformer {
 
         response.aggregationPayload = this.getAggregatePayload(request, data);
 
-        response.payload = data
-            .slice(request.skip, request.skip + request.take)
-            .map((row: Record<string, unknown>) => parsePayload(row, request.columns));
+        if (request.take > 0) {
```

Comment:
Please refactor to avoid if/else statement. If you look closely, you'll realize you're duplicating almost 3 LOC. Lines 31 and 45 can be merged into a single one so that you can avoid the if/else.

Reply:
Got it, let me fix that